packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md

```diff
@@ -0,0 +1,53 @@
+After performing [local signout](/docs/guides/sign-users-out/react/sign-out-of-your-app/), navigate the user's browser to the [OIDC logout page](https://developer.okta.com/docs/reference/api/oidc/#logout).
+
+This page will clear the user's Okta session then redirect back to the `post_logout_redirect_uri` provided. This URI must be one of those listed in the `Logout redirect URI` section of your application's settings.
+
+Open your Okta Developer Console:
+
+<a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
+
+1. Select **Applications**, and then pick your application.
+
+2. Select **General** and click **Edit**.
+
+3. In the **Logout redirect URI section**, add the **Base URI** of your application, optionally followed by a path, for example, `http://localhost:8080/logged_out`. Also, add any URIs where your application runs in production, such as `https://app.example.com/logged_out`.
```

Comment:
3. In the **Logout redirect URI section**, add the **Base URI** of your application. You can optionally follow that URI with a path, for example, `http://localhost:8080/logged_out`.